tests/drivers/sdhc/src/main.c

```diff
@@ -9,7 +9,7 @@
 #include <zephyr/device.h>
 #include <ztest.h>
 
-static const struct device *sdhc_dev;
+static const struct device *sdhc_dev = DEVICE_DT_GET(DT_NODELABEL(sdhc0));
```

Comment:
as said, require aliases instead.

Reply:
Will rework to use an alias.  Any particular alias name/convention we are using for tests?

Reply:
`sdhc0` could be an alias, similar to e.g. `magn0`, `led0`, etc.